src/org/labkey/test/BaseWebDriverTest.java

```diff
@@ -1141,6 +1143,8 @@ private void doPostamble()
         if (isTestRunningOnTeamCity())
             checkActionCoverage();
 
+        CspLogUtil.checkNewCspWarnings(getArtifactCollector());
```

Comment:
Since `CspLogUtil.checkNewCspWarnings` can throw an assert or an exception shouldn't this line be in a try/catch? Otherwise this is saying that if there was some issue with the csp we cannot/will not check the links or any of the other actions that follow.

Reply:
This matches the behavior for all of the existing checks